tests/console/commands/test_publish.py

```diff
@@ -14,10 +15,10 @@ def test_publish_returns_non_zero_code_for_upload_errors(app, app_tester, http):
  - Uploading simple-project-1.2.3.tar.gz 0%
  - Uploading simple-project-1.2.3.tar.gz 100%
  - Uploading simple-project-1.2.3.tar.gz 100%
-                               
```

Comment:
Looks like you removed whitespace here - was that intentional? Might be related to the test failure.

Reply:
oh, I totally did. Don't know why I didn't notice that. But it explains my test failure. I'll correct that.